### ISSUE TITLE
Interpreter has problem with /extern function definition if lit-word is used too

### ISSUE THREAD

**Issue by Oldes** [red/red#2539]
This code:
```
buffer: copy ""
test: function [
	'word [any-type!]
	/extern buffer
][]
```
compiles fine, but fails in console (interpreted):
```
*** Script Error: invalid /extern value: 'word
*** Where: function
```

**Comment by dockimbel** [comment]
Good catch!

**Comment by dockimbel** [comment]
Fixed by PR #2540.
